### PR TITLE
Enable cloudtrail elasticco for landing account

### PR DESCRIPTION
**Pull request by tomskelley-gjs**

### CODE REVIEW

File: cloud-trail-local.tf [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        },
+        "StringLike": {
+          "kms:EncryptionContext:aws:cloudtrail:arn": "arn:aws:cloudtrail:*:${data.aws_caller_identity.current.account_id}:trail/*"
+        }
+      }
+    }
+  ]
+}
+POLICY
+}
```
Comment (damacus) [inline cloud-trail-local.tf:8]: I'd suggest moving from incline policies to `aws_iam_policy_document` before we apply this one. They're much better to read and use interpolation with going forward.

File: cloud-trail-local.tf [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+# user module froom analytics-platform-ops
+module "aws_account_logging" {
+  source = "modules/aws_account_logging"
+
+  es_domain   = "${var.es_domain}"
+  es_port     = "${var.es_port}"
+  es_scheme   = "${var.es_scheme}"
+  es_username = "${var.es_username}"
+  es_password = "${var.es_password}"
```
Comment (damacus) [inline cloud-trail-local.tf:99]: As we discussed on Slack. These sensitive variables should either be sourced from data_sources (from elastic) or held in SecretsManager.

File: modules/aws_account_logging/cloudtrail/.gitignore [hunk 3]
```diff
@@ -0,0 +1,9 @@
+*.dist-info
+*.egg-info
+certifi
+elasticsearch
+requests
+s3_log_parser
+ua_parser
+urllib3
+user_agents
```
Comment (damacus) [inline modules/aws_account_logging/cloudtrail/.gitignore:9]: Probably worth moving this to the main gitignore.

**Review by damacus — CHANGES_REQUESTED** [review]

File: cloud-trail-local.tf (hunk 2 above)
Comment (tomskelley-gjs) [reply cloud-trail-local.tf:99]: One for later I think.

File: modules/aws_account_logging/cloudtrail/.gitignore (hunk 3 above)
Comment (tomskelley-gjs) [reply modules/aws_account_logging/cloudtrail/.gitignore:9]: Added to main gitignore but I’m not removing it as this should be a vendored module later on and I want to keep it the same everywhere for now.

**Review by damacus — APPROVED** [review]

**Pull request merged**